app/graphql/context.js

```diff
@@ -1,8 +1,10 @@
 import { RuralPaymentsAgencyLandAPI } from '../data-sources/rural-payments-agency-land-api.js'
 import { RuralPaymentsPortalApi } from '../data-sources/rural-payments-portal/RuralPaymentsPortalApi.js'
+import { getADGroups } from '../auth/authenticate.js'
 
-export function context () {
+export async function context ({ request }) {
```

Comment:
This is a circular dependency, you need to create a plugin for authorisation and execute the context adGroup